pkg/operator/ceph/pool/controller.go

```diff
@@ -385,13 +385,16 @@ func createPool(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo,
 		return errors.Wrapf(err, "failed to create pool %q", p.Name)
 	}
 
-	logger.Infof("initializing pool %q", p.Name)
+	if appName != poolApplicationNameRBD {
+		return nil
+	}
```

Comment:
Is this the right behavior? Should we instead return a request to requeue the reconcile in 5-15 seconds or something instead so we are sure to configure the pool once it is initialized? @travisn ?

Reply:
For non-rbd pools, the pool configuration is done at this point. For these pools there is no need to run `rbd pool init`, so we can just return.